crates/cxx-qt-lib/build.rs

```diff
@@ -251,5 +251,52 @@ fn main() {
     });
     println!("cargo:rerun-if-changed=src/assertion_utils.h");
 
-    builder.with_opts(cxx_qt_lib_headers::build_opts()).build();
```

Comment:
this makes `cxx-qt-lib` worse to build as we need to duplicate the header copy phase :-/